### PR TITLE
Added language Central Kurdish (Soranî)

### LINKED ISSUE #2528: Kurdish (Sorani) (ckb)
* Language name: Kurdish (Sorani) 
* Language info: https://en.wikipedia.org/wiki/Sorani
* ISO 639-3 code: CKB
* Public List: https://tatoeba.org/eng/sentences_lists/show/168673
* Icon: https://imgur.com/a/pIMJJW6
* Requested by: https://tatoeba.org/eng/user/profile/Sarchia

Note: We're waiting members to tell us if their sentences are in this language.

### PR DESCRIPTION
**Pull request by RyckRichards**
This PR closes #2528.
It adds Kurdish (Sorani), one of the languages that will replace Kurdish (Kur) since it's a macrolanguage.

### CODE REVIEW

File: src/Lib/LanguagesLib.php [hunk 1]
```diff
@@ -612,6 +612,7 @@ public static function languagesInTatoeba()
                 'mni' => __d('languages', 'Meitei'),
                 'ayl' => __d('languages', 'Libyan Arabic'),
                 'ood' => __d('languages', "O'odham"),
+                'ckb' => __d('languages', 'Kurdish (Sorani)',
```
Comment (jiru) [inline src/Lib/LanguagesLib.php:615]: You forgot a closing parenthesis.

**Comment by trang** [comment]
@RyckRichards Please change the name to `Central Kurdish (Sorani)` as suggested in https://github.com/Tatoeba/tatoeba2/issues/2529#issuecomment-699597267.

**Comment by RyckRichards** [comment]
Sorry for taking too long. I've commited the changes.

File: src/Lib/LanguagesLib.php [hunk 2]
```diff
@@ -612,6 +612,7 @@ public static function languagesInTatoeba()
                 'mni' => __d('languages', 'Meitei'),
                 'ayl' => __d('languages', 'Libyan Arabic'),
                 'ood' => __d('languages', "O'odham"),
+                'ckb' => __d('languages', 'Central Kurdish (Soranî)',
```
Comment (AndiPersti) [inline src/Lib/LanguagesLib.php:615]: The closing parenthesis is still missing on this line (before the last comma).

Comment (RyckRichards) [reply src/Lib/LanguagesLib.php]: Thank you!

**Pull request merged**